src/rt/cover.c

```diff
@@ -19,48 +19,68 @@
 #include "array.h"
 #include "common.h"
 #include "cover.h"
-#include "diag.h"
-#include "fbuf.h"
-#include "hash.h"
 #include "lib.h"
 #include "opt.h"
 #include "type.h"
+#include "rt.h"
+#include "model.h"
 
 #include <assert.h>
 #include <stdlib.h>
 #include <string.h>
 #include <limits.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <unistd.h>
+#include <time.h>
 
-#define SUB_COND_BITS 5
-#define MAX_SUB_CONDS (1 << SUB_COND_BITS)
-#define SUB_COND_MASK (MAX_SUB_CONDS - 1)
+//#define COVER_DEBUG
 
-#define PERCENT_RED    50.0f
-#define PERCENT_ORANGE 90.0f
+#define MARGIN_LEFT "400"
+#define SIDEBAR_WIDTH "370"
```

Comment:
OK, I have put it to 30% fixed always. I have tried to experiment with CSS `clamp` function, but
it gave me weird behavior. I don't think it matters that much as long as the UI is usable. Also,
I don't really like front-end web stuff, so I try to avoid it...